server/database/queries/User/checkUserByEmail.js

```diff
@@ -0,0 +1,14 @@
+const connection = require('../../config/connection');
+
+const checkEmail= (email)=> {
+    const sqlQuery = {
+        text: 'SELECT * FROM users WHERE email = $1;',
```

Comment:
You are only checking if the user exists or not, so you only need to return the id instead of a lot of unused data

Reply:
This query is used for signin and signup, for sign in, you'll need to get user data